sdk/keyvault/Azure.Security.KeyVault.Administration/tests/AccessControlTestBase.cs

```diff
@@ -14,9 +16,18 @@ namespace Azure.Security.KeyVault.Administration.Tests
     public class AccessControlTestBase : RecordedTestBase<KeyVaultTestEnvironment>
     {
         public KeyVaultAccessControlClient Client { get; set; }
+        public KeyClient Key_Client { get; set; }
 
         public Uri VaultUri { get; set; }
 
+#pragma warning disable IDE1006 // Naming Styles
```

Comment:
Any reason you can't just follow the style for consistency? You can change all instances in a doc with Ctrl+., select Document (or solution, even), and Apply.

Reply:
At one point, these are exposed in the samples and the naming was intentional. Now only `client` is exposed, so fixing all but that one.